style.css

```diff
@@ -251,8 +251,9 @@ ul.card-item {
 }
 
 .card-lists h3 {
-  text-align: center;
-  font-size: 30px;
+  /* text-align: center;
+  font-size: 30px; */
```

Comment:
- [ ] To make your code clean...
Please, clear your project from unused commented codes!

Reply:
done

Reply:
done